1-exercises/E-array-map/exercise.js

```diff
@@ -3,11 +3,15 @@
 
 let numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
-let numbersMultipliedByOneHundred; // complete this statement
+// let numbersMultipliedByOneHundred = numbers.map((number) => number * 100);
+
+let numbersMultipliedByOneHundred = numbers.map(function (number) {
```

Comment:
It could be a `const` instead of a `let`, by the way. It's better to make sure it's not mistakenly changed later if you don't indent to.